packages/block-library/src/post-template/edit.js

```diff
@@ -123,7 +123,7 @@ export default function PostTemplateEdit( {
 					slug: templateSlug.replace( 'category-', '' ),
 				} );
 			const query = {
-				offset: perPage ? perPage + offset : 0,
+				offset: perPage ? offset || 0 : 0,
```

Comment:
`|| 0` ensures that the value of the `offset` property is always a number. Without this code, when you empty the offset field, the value of the offset property will be an empty string and the spinner will remain visible.

https://github.com/WordPress/gutenberg/assets/54422211/318626f5-517f-4d45-8a0b-3935b1027d6a

Reply:
Good point, I updated 👍